quartz_solar_forecast/data.py

```diff
@@ -37,16 +38,39 @@ def get_gfs_nwp(site: PVSite, ts:datetime) -> xr.Dataset:
     start = ts.date()
     end = start + pd.Timedelta(days=7)
 
-    # Getting GFS, from OPEN METEO
+    # Getting NWP, from OPEN METEO
+    urlStart = None
+    if source == "icon":
+        urlStart = "dwd-icon"
+    if source == "gfs":
+        urlStart = "gfs"
+
```

Comment:
you could do a `if` and `elif` and `else`. SO it would be somthing like
```
if source == "icon":
     urlStart = "dwd-icon"
elif source == "gfs":
    urlStart = "gfs"
else:
    rase Exception(f'Source ({source}( must be either icon or gfs')